### ISSUE TITLE
Add pure-CSS Float Label demo

### ISSUE THREAD

**Issue by anton-staroverov** [una/YouMightNotNeedJS#28]
A good addition to the YouMightNotNeedJS will be a pure-CSS Float Labels:
https://github.com/tonystar/float-label-css

**Comment by robsonsobral** [comment]
Bad accessibility support, since the label of the field has to come after it. This should be flagged.

**Comment by anton-staroverov** [comment]
@robsonsobral there is alternative style which is also supported:

```html
<label class="has-float-label">
  <input type="email" placeholder="email@example.com"/>
  <span>Email</span>
</label>
```

https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly
